portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -27,13 +27,15 @@
 import com.google.appengine.api.datastore.PreparedQuery;
 import com.google.appengine.api.datastore.Query;
 import com.google.appengine.api.datastore.Query.SortDirection;
+import com.google.Constants;
+
 
 /** Servlet that returns bucket list content */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
   private static final String COMMENT_PARAMETER = "comment-area";
-  private static final String LIST_ITEM = "listItem";
+  private static final String ENTITY_KIND = "listItem";
```

Comment:
Once Constants.java is working, you can remove this and then replace any references to it with `Constants.ENTITY_KIND`